### ISSUE TITLE
KeyBase is DEAD

### ISSUE THREAD

**Issue by DogsAndTwelveKittens** [keybase/client#24577]
I’m calling on the Contributors to confess this, and allow people to move on- since the acquisition by Zoom this GitHub has literally been quieter by ever before by a HUGE margin… the two and a half people left working on KeyBase seem to do nothing as is.

**Comment by bytefend** [comment]
Agreed. I feel as if it's a security issue at this point when nobody is pushing updates reguarly, there's one update every few months.

**Comment by Rudi9719** [comment]
@bytefend  have there been any vulnerabilities found in the Keybase client recently that need patching? Otherwise I'm not sure how there could be a security issue from lack of patches. @DogsAndTwelveKittens what are you expecting the contributors to confess? Do you feel like they're hiding anything from you or lying to you?

**Comment by bytefend** [comment]
I have multiple 0days for keybase @Rudi9719

**Comment by Rudi9719** [comment]
> I have multiple 0days for keybase @Rudi9719 

These should be reported immediately to the Zoom Bug Bounty page then. https://hacker one.com/zoom

**Comment by kiwimato** [comment]
I agree with OP, to me it sounds almost dead, looking at [this](https://github.com/keybase/client/issues/24413) which basically makes Keybase uninstallable on newer Debian/Ubuntu flavors makes me think it's close to being unmaintained.
Basically Keybase ["how-to install"](https://keybase.io/docs/the_app/install_linux) documentation for Ubuntu/Debian flavor doesn't work and seems nobody cares.
After a quick look at 1st page of issues seems like nobody is actively involved in the project anymore. A look at the commit page I see few devs who are active, so maybe it's just an issue that it's they're simply not enough devs and too many issues?
I would love to contribute but not sure who to talk to, especially after being bought by Zoom.

**Comment by kiobu** [comment]
Time to start using [keys.pub](https://keys.pub/) I guess.

**Comment by BNolet** [comment]
From the blog post it sounded like they planned to continue working on KeyBase... if that's the case it's a damn shame as that would mean that KeyBase was literally just acquired for the technology and engineering

**Comment by Rudi9719** [comment]
[New Release](https://github.com/keybase/client/releases/tag/v5.8.1)

**Comment by orgcontrib** [comment]
A year earlier:

- https://github.com/keybase/client/issues/24303

**Comment by piotr-yuxuan** [comment]
<img width="1196" alt="Screenshot 2022-01-24 at 11 57 36" src="https://user-images.githubusercontent.com/3686502/150779042-a32102ca-b1ad-4b9e-b00f-c787b128f446.png">

It is dead indeed; time to move on. Would you advice for any alternative tooling?

**Comment by caesar** [comment]
> Would you advice for any alternative tooling?

It's a bit less polished / requires more technical understanding to use, but [Keyoxide](https://keyoxide.org/) is great, and as it's decentralised it doesn't rely on any single party who could shut it down.

**Comment by junderw** [comment]
I wonder if zoom is willing to sell off the keybase service and a few of the lead engineers so that some other company could actually do something with keybase?

**Comment by virtadpt** [comment]
Doubtful.  I think they're trying to kill it through neglect, rather than just shutting everything down.

**Comment by Southclaws** [comment]
![image](https://github.com/keybase/client/assets/1636971/4c0c8d46-e145-41bb-81c0-62194e040002)

Nail in the coffin?

**Comment by Cryptophobia** [comment]
TLDR: **Don't use Keybase. It is unsupported, insecure (since 2023), and unmaintained.** 

**PSA:**
The feds killed Keybase by advising Zoom to buy it after Keybase started the crypto wallet support and Zoom had end-to-end encryption problems in 2020 during the COVID lockdowns. Coincidentally, this is also when Keybase was doing the XLM (Stellar) drops and supporting beginning to work with crypto wallets on its platform. Keybase was probably threatened by the feds and forced to sell to Zoom due to this crypto wallet endeavor. I also doubt Zoom was even remotely interested in buying Keybase until it was advised by the feds entities to acquire their security team in order to fix the encryption issues. 

You just have to follow the news to figure this out: 

1) May 2019: Keybase first added support for a built‑in cryptocurrency wallet in May 2019, when it enabled Stellar (XLM) wallet functionality for all Keybase users.

2) April 2020: Zoom faced major encryption and privacy issues during the early months of the COVID-19 pandemic, particularly in March and April 2020, when its usage skyrocketed due to the global shift to remote work and learning.

3) May 2020:  Zoom acquired encryption company Keybase to develop end-to-end encryption. (Most likely based on advisory services of the feds after forensics investigations.)

4) June 2020: Contribution graph for Keybase dies.

5) October 2023: Keybase officially ended wallet support for Stellar (XLM) transactions.

Keybase could have become the largest platform for public-key service and crypto wallet services for developers worldwide. However, it is not in the interest of the feds to have Keybase compete with Github or other platforms as GitHub is part of MSFT and MSFT does everything the feds want it to.

**Comment by Southclaws** [comment]
source: i made it up 🥀

**Comment by Cryptophobia** [comment]
> source: i made it up 🥀

Logical interpretation of facts. If you have another interpretation why Keybase is dead after being so widely used before it was acquired, I like to hear it. 

Zoom did not even try to build a service to replace it or change how the service worked. No major new releases.

**Comment by vraravam** [comment]
As long as the service is active (ie kept alive) and any security fixes are promptly put in place and released to the general users - I would be ok with that!
I mainly use the private e2e encrypted git repos - and if the service shuts down, I would need to look for alternatives!

**Comment by Southclaws** [comment]
>If you have another interpretation why Keybase is dead after being so widely used 

standard M&A acqui-hire for talent, happens all the time, ain't no "fed" conspiracy unless you have sources...

**Comment by Cryptophobia** [comment]
> > If you have another interpretation why Keybase is dead after being so widely used
> 
> standard M&A acqui-hire for talent, happens all the time, ain't no "fed" conspiracy unless you have sources...

You seriously believe this or just talking without any brain cells left?

If that was the case, then why did the founder and CTO of Keybase already leave Zoom? I'm happy for the Keybase founder and CTO getting a payout that they deserve, but there is no point lying to yourself on a Github thread buddy.

Maxwell Krohn

> Head of Security Engineering
> Zoom · Full-time May 2020 - Jan 2025 · 4 yrs 9 mos
> May 2020 to Jan 2025 · 4 yrs 9 mos 
> New York City Metropolitan Area

Source: LinkedIn

Apparently critical thinking and logic is not part of your skill set.

Source: Above posts.

**Comment by vraravam** [comment]
There's no need to insult another person imho.

The founder could have left for any number of reasons. In the reverse, do you have proof about why the found and CTO quit zoom? Maybe they were of the mindset to retire? or maybe they have been brewing a new venture? maybe health? any number of reasons could be there. Without proof, no one should cause FUD and propaganda imho.

**Comment by iminto** [comment]
what a pity,RIP
